src/Models/Message.php

```diff
@@ -38,6 +39,16 @@ public function ticket(): BelongsTo
         );
     }
 
+    /**
+     * Get User RelationShip
+     *
+     * @return \Illuminate\Database\Eloquent\Relations\BelongsTo
+     */
+    public function user(): BelongsTo
+    {
+        return $this->belongsTo(User::class);
```

Comment:
Can you please follow the pattern like the rest of the methods? Like the `ticket()` method, and adding the foreign key to the config file under `messages` key will be appreciated!

If you can add few tests for the `user()` method under `MessageTest` file, will be perfect, for future changes.